core/templates/dev/head/pages/community-dashboard-page/question-opportunities/question-opportunities.directive.html

```diff
@@ -0,0 +1,7 @@
+<div>
+  <opportunities-list data-loading="$ctrl.opportunitiesAreLoading"
```

Comment:
Why this PR have changes related to listing opportunities? (similar changes in #7602)

Reply:
This PR branches off of #7602 since it needs to interact with SkillOpportunity objects